tests/spread/integration/erlang/task.yaml

```diff
@@ -0,0 +1,27 @@
+summary: Integration tests for Erlang
+
+execute: |
+  rootfs="$(install-slices erlang-base_bins erlang-inets_scripts erlang-os-mon_bins erlang-crypto_modules erlang-eldap_modules erlang-mnesia_modules erlang-parsetools_modules erlang-public-key_modules erlang-runtime-tools_modules erlang-ssl_modules erlang-syntax-tools_modules erlang-tools_modules erlang-xmerl_modules base-files_base bash_bins coreutils_bins)"
+
+  mkdir -p "$rootfs"/bin "$rootfs"/etc "$rootfs"/dev "$rootfs"/root
+  cp /bin/sh "$rootfs"/bin/
+  cp /etc/resolv.conf "$rootfs"/etc/
+  cp /etc/nsswitch.conf "$rootfs"/etc/
+  touch "$rootfs"/dev/null
+  chmod 777 "$rootfs"/dev/null
+  echo "cookie" > "$rootfs"/root/.erlang.cookie
+  chroot "$rootfs" chmod 400 /root/.erlang.cookie
```

Comment:
I think this block could have gone on to the `prepare:` section.

In any case, let's please use empty lines to separate logical blocks. Easier to read and follow.

Reply:
we can't really use `prepare:` here because we won't be able to use the $rootfs env var in `execute:` then.